src/applications/static-pages/wizard/index.js

```diff
@@ -3,16 +3,40 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import set from 'platform/utilities/data/set';
 
+export const NO_BENEFIT_REFERRED = 'no benefit was referred';
+export const WIZARD_STATUS_NOT_STARTED = 'not started';
+export const WIZARD_STATUS_COMPLETE = 'complete';
+export const WIZARD_STATUS_APPLY_NOW = 'awaiting click on apply button';
+export const WIZARD_STATUS_IN_PROGRESS = 'in progress';
+export const WIZARD_STATUS_UPDATING = 'updating';
+export const formIdSuffixes = {
+  FORM_ID_1990: '1990',
+  FORM_ID_10203: '10203',
+  FORM_ID_1995: '1995',
+  FORM_ID_0994: '0994',
+  FORM_ID_5495: '5495',
+  FORM_ID_5490: '5490',
+  FORM_ID_1990E: '1990E',
+  FORM_ID_1990N: '1990N',
+};
+
+export const getReferredBenefit = async () =>
+  (await sessionStorage.getItem('benefitReferred')) || NO_BENEFIT_REFERRED;
+
+export const getWizardStatus = async () =>
+  (await sessionStorage.getItem('wizardStatus')) || WIZARD_STATUS_NOT_STARTED;
+
 export default class Wizard extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       pageHistory: [props.pages[0]],
       currentPageIndex: 0,
       expanded: !props.expander,
+      benefitReferred: getReferredBenefit(),
+      wizardStatus: getWizardStatus(),
     };
```

Comment:
`benefitReferred` should be more generic (have a more generic name) to suit all services that use the wizard.